### ISSUE TITLE
Purchases: Fix alignment of WordPress logo on free credit screen

### ISSUE THREAD

**Issue by jeremeylduvall** [Automattic/wp-calypso#12928]
This is another small style issue that should be easy to address. On the checkout screen for a domain when you have a free domain credit, the WordPress logo should be aligned vertically in the middle of the screen.

#### Steps to reproduce
1. Make sure you're on an account with a domain credit available.
2. Go through the steps to purchase a domain.
3. When you get to the checkout page, you should see the "You have a free domain credit!" message with the alignment issue.

<img width="724" alt="screen shot 2017-04-08 at 11 28 02" src="https://cloud.githubusercontent.com/assets/7240478/24830576/7a92e366-1c4e-11e7-8e07-8027f861f76a.png">

**Comment by hd719** [comment]
Hello I am new to open source can I work on this?

**Comment by jeremeylduvall** [comment]
@hd719 Definitely - feel free to work on this and submit a PR when it's ready. If you haven't checked it out yet, we have some guidelines around creating PRs here:

https://github.com/Automattic/wp-calypso/blob/master/.github/CONTRIBUTING.md

You'll need a domain credit on your account to test this out (since the issue only occurs with a free domain credit). It looks like you have a WordPress.com account already. I'll reach out through that email regarding that credit.

**Comment by hd719** [comment]
Hey @jeremeylduvall real quick question (just to make sure I understand the issue). 

![screen shot 2017-04-24 at 9 15 43 am](https://cloud.githubusercontent.com/assets/20329318/25338843/a0813b9a-28ce-11e7-8dd9-b64437fecd33.png)

You want the Word Press Logo to be centered vertically in the box (where is shows "You have a free domain credit"), correct? 

-Thank you.

**Comment by hd719** [comment]
Hey @jeremeylduvall so I was able to center the Word Press Logo using the code above 

![screen shot 2017-05-01 at 7 41 56 pm](https://cloud.githubusercontent.com/assets/20329318/25598968/77c61ca6-2ea6-11e7-9abe-2a7ece6b18af.png)

But the only problem I am having now is this: 

![screen shot 2017-05-01 at 7 42 14 pm](https://cloud.githubusercontent.com/assets/20329318/25598976/81d27c26-2ea6-11e7-9ea1-1e200058b91e.png)

This is the CSS code I am using 

```
.credits-payment-box {
		.payment-box-section {
			// box-sizing: border-box;
			// min-height: 91px;
			// padding: 20px 20px 20px 80px;
			// position: relative;

			display: flex;
			flex-direction: column;
			justify-content: center;
			
			
			&::before {
				color: $blue-medium;
				position: absolute;
				text-align: center;
				// width: 100px;
				// height: 100px;
				//top: 50px;
				left: 10px;
				// top: 25px; 
				// left: 50px; 
				// margin-left: -50px;
				text-align: center;
				max-width: 50%;
				@include noticon( '\f205', 60px );
				@include breakpoint( ">660px" ) {
					left: 50px;
				}
			}
```


-Thank you.

**Comment by jeremeylduvall** [comment]
Closed in #13460. h/t @hd719!
